code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -30,7 +30,7 @@
 	screamsound = list('sound/voice/ethereal/ethereal_scream_1.ogg', 'sound/voice/ethereal/ethereal_scream_2.ogg', 'sound/voice/ethereal/ethereal_scream_3.ogg')
 	sexes = FALSE //no fetish content allowed
 	toxic_food = NONE
-	inert_mutation = SHOCKTOUCH
+	inert_mutation = SHOCKTOUCH,OVERLOAD
```

Comment:
you'll have to refactor inert_mutations to accept lists or remove one of them as it currently only supports 1 mutation